src/script/interpreter.cpp

```diff
@@ -288,9 +289,8 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
     // static const valtype vchZero(0);
     static const valtype vchTrue(1, 1);
 
-    CScript::const_iterator pc = script.begin();
-    CScript::const_iterator pend = script.end();
-    CScript::const_iterator pbegincodehash = script.begin();
+    Span<const unsigned char> pc = script;
+    Span<const unsigned char> codehash = script;
     opcodetype opcode;
     valtype vchPushValue;
```

Comment:
With the changes, `vchPushValue` is now unused and can be deleted.

Reply:
Nice find. Gone.